velox/docs/functions/spark/string.rst

```diff
@@ -55,6 +55,18 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT endswith('js SQL', 'js'); -- false
         SELECT endswith('js SQL', NULL); -- NULL
 
+.. spark:function:: find_in_set(str, strArray) -> integer
+
+    Returns the index (1-based) of the given string ``str`` in the comma-delimited list ``strArray``.
+    Returns 0, if the string was not found or if the given string ``str`` contains a comma. ::
+
+        SELECT find_in_set('ab', 'abc,b,ab,c,def'); -- 3
+        SELECT find_in_set('', ''); -- 1
```

Comment:
> SELECT find_in_set('', ''); -- 1

This behavior is surprising. Perhaps, document it explicitly. Is there a unit test to verify this?